R/easieR.R

```diff
@@ -658,7 +658,7 @@ ref1 <-
   packageStartupMessage(desc_first_time_easier)
   packageStartupMessage(desc_special_characters_have_been_removed)
   packageStartupMessage(textVersion)
-  packageStartupMessage("Last update 06/05/2023") # TODO translation
+  packageStartupMessage("Last update 03/12/2024") # TODO translation?
```

Comment:
I do not think it is necessary. 
The aim is to determine the version installed by users when a problem occurs